code/modules/goonhub/api/routes/RandomEntries.dm

```diff
@@ -5,4 +5,4 @@
 	method = RUSTG_HTTP_METHOD_GET
 	path = "/random-entries"
 	queryParams = list("type", "count")	// not sure if i should put 0 or "count" here.
-	correct_response = /datum/apiModel/randomEntries	//this model/resource/schema doesn't exist
+	correct_response = /datum/apiModel/Tracked/EventTicketResource // this looks correct? best to double check it
```

Comment:
it can be multiple things - 
![image](https://github.com/goonstation/goonstation/assets/4741640/11f40e9c-2e52-4e72-9bbd-d49cd9854e79)

plus more

probably should be represented as a list of the typepaths?

Reply:
i've added the four shown here as the docs don't appear to list any further ones. In the form of a list of typepaths. if more need to be added then just let me know